Sequence of Tenses/maxi.md

```diff
@@ -0,0 +1,39 @@
+# Please translate the following sentences into English
+
+1. На прошлой неделе Сергей заметил, что его коллега по проекту пользуется готовыми снипетами, что ускоряло его работу, но видимо он передумал и решил писать код по старинке.
+
+On the previous week, Sergiy noticed, that his colleague on the project using the prepared snippet of code, that increased the speed of work, but he changed the mind and decided writing the code as usual 
```

Comment:
at the previous weel/ last week
was using the ready-made
decided to write - we use infinitive after decide https://www.engvid.com/english-resource/verbs-followed-by-gerunds-and-infinitives/